sky/packages/sky/lib/base/scheduler.dart

```diff
@@ -16,6 +16,7 @@ final List<Callback> _persistentCallbacks = new List<Callback>();
 Map<int, Callback> _transientCallbacks = new LinkedHashMap<int, Callback>();
 final Set<int> _removedIds = new Set<int>();
 
+/// Called by the engine to being an animation frame.
```

Comment:
You should clarify this comment further. This method is somewhat key to the whole system and deserves more detail.